src/wp-includes/functions.php

```diff
@@ -1899,7 +1899,8 @@ function wp_nonce_field( $action = -1, $name = '_wpnonce', $referer = true, $ech
  * @return string Referer field HTML markup.
  */
 function wp_referer_field( $echo = true ) {
-	$referer_field = '<input type="hidden" name="_wp_http_referer" value="' . esc_attr( wp_unslash( $_SERVER['REQUEST_URI'] ) ) . '" />';
+	$request_url   = remove_query_arg( '_wp_http_referer' );
```

Comment:
shouldn't this pass in the `$_SERVER['REQUEST_URI']` or is the fallback current url ok here?

Reply:
remove_query_arg calls add_query_arg and that uses $_SERVER['REQUEST_URI'] is it is missing 
So think this is ok

Reply:
It was removed as per https://github.com/WordPress/wordpress-develop/pull/2242#discussion_r981722715 review comment by David.